code/modules/vehicles/mecha/combat/gygax.dm

```diff
@@ -98,9 +98,24 @@
 	. = ..()
 	max_ammo()
 
-/obj/vehicle/sealed/mecha/gygax/dark/add_cell(obj/item/stock_parts/cell/C=null)
-	if(C)
-		C.forceMove(src)
-		cell = C
+/obj/vehicle/sealed/mecha/gygax/dark/add_cell(obj/item/stock_parts/cell/power_cell)
+	if(power_cell)
+		power_cell.forceMove(src)
+		cell = power_cell
 		return
 	cell = new /obj/item/stock_parts/cell/bluespace(src)
+
+/obj/vehicle/sealed/mecha/gygax/dark/add_scanmod(obj/item/stock_parts/scanning_module/sm)
```

Comment:
scanner(?) or scanmod
if you name it scanmod you will need to change L113 to src.scanmod. This is best practice

please avoid abbreviations

Reply:
scanner?

Reply:
I dont think theres a need to change sm, because to me its kinda clear that sm - isnt supermatter or soy milk

Reply:
uh, fine, its still doesnt quite matter

Reply:
you just named two different abbreviations that this couldve been then said it was fine